### PR TITLE
tarball uses hermetic coreutils

### PR DESCRIPTION
**Pull request by rickvanprim**
Updates `tarball.sh.tpl` to use the hermetic coreutils.  I opted to match the existing script form of supplying a variable for the executable instead of the newer `image.sh.tpl` form of modifying `PATH` and assuming the binary name (particularly as updating `tar` doesn't seem like it would work as the binary name is different between Windows and Unix).  If you'd prefer I modify `PATH` to find `coreutils`, I could do that instead.

### CODE REVIEW

File: oci/private/tarball.sh.tpl [hunk 1]
```diff
@@ -17,14 +18,14 @@ readonly OUTPUT="{{output}}"
 function add_to_tar() {
     content=$1
     tar_path=$2
-    echo >>"${OUTPUT}" "${tar_path} uid=0 gid=0 mode=0755 time=1672560000 type=file content=${content}"
+    "${COREUTILS}" echo >>"${OUTPUT}" "${tar_path} uid=0 gid=0 mode=0755 time=1672560000 type=file content=${content}"
```
Comment (thesayyn) [inline oci/private/tarball.sh.tpl:21]: isn't echo shell built-in? i think we don't need coreutils for this.

Comment (rickvanprim) [reply oci/private/tarball.sh.tpl:21]: Fair, I will remove that.  I was thinking it wasn't builtin to `sh` but seems I'm foolishly wrong and it's builtin everywhere: https://unix.stackexchange.com/a/1359

Comment (rickvanprim) [reply oci/private/tarball.sh.tpl]: Updated PR.  Busy day 😅

**Review by thesayyn — APPROVED** [review]

**Comment by thesayyn** [comment]
@rickvanprim there seems to be some conflicts? can you resolve those?

**Comment by rickvanprim** [comment]
> @rickvanprim there seems to be some conflicts? can you resolve those?

Done.  That was weird, the merge conflict was in `load.bzl` which I hadn't touched 🤷‍♂️

**Comment by rickvanprim** [comment]
Ah the file got renamed, that makes sense 😅

**Pull request merged**